pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixInstanceDataManager.java

```diff
@@ -180,6 +181,24 @@ private void initInstanceDataDir(File instanceDataDir) {
     }
   }
 
+  @Override
+  public List<File> getConsumerDirPaths() {
+    String consumerDirPath = _instanceDataManagerConfig.getConsumerDir();
+
+    if (consumerDirPath != null) {
+      return Collections.singletonList(new File(consumerDirPath));
+    } else {
+      List<File> consumerDirs = new ArrayList<>();
```

Comment:
This code block is repeating the logic in RealtimeTableDataManager.getConsumerDir right? Can we move that logic to a common place and reuse from both places? 

Duplicating logic can result in the metrics breaking if this logic changes in`RealtimeTableDataManager.getConsumerDir`

Reply:
done